packages/one-app-server-bundler/__tests__/utils/patchedCryptoHash.spec.js

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2023 American Express Travel Related Services Company, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations
+ * under the License.
+ */
+const crypto = require('crypto');
```

Comment:
This may not be required, I believe webpack 5 fixed this issue

Reply:
Will look into that

Reply:
<img width="1116" alt="image" src="https://github.com/americanexpress/one-app-cli/assets/2575450/946ca7c3-a875-4a7d-9e92-9feed36e4fa3">
Without it crypto throws when an md5 is requested. Our behaviour is to return a sha256 when md5 is requested.